docs-svelte-kit/build-system/build.mts

```diff
@@ -32,13 +33,14 @@ function build(input: string, out: string, injects: string[] = []) {
 
 /** bundle */
 function bundle(entryPoint: string, externals: string[]) {
+  const injectPath = path.join(dirname, "./src/process-shim.mjs")
   const result = esbuild.buildSync({
     entryPoints: [entryPoint],
     format: "esm",
     bundle: true,
     external: externals,
     write: false,
-    inject: [path.join(dirname, "./src/process-shim.mjs")],
+    inject: [injectPath],
```

Comment:
This change seems unnecessary?

Reply:
Yeah. I used it for debugging and forgot to put it back 😓